dev.mateusz1913.Pilot.json

```diff
@@ -0,0 +1,58 @@
+{
+    "id" : "dev.mateusz1913.Pilot",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "pilot",
+    "finish-args" : [
+        "--share=network",
```

Comment:
why does this need network?

Reply:
It's needed in order to connect and communicate with AndroidTV emulators and devices via [`adb`](https://developer.android.com/tools/adb) - when developing Android apps, we can connect the devices to the host machine via WiFi. It's also needed for connecting to emulators - I tried to remove the permission for testing purposes and after removal it doesn't recognize any running emulator (`adb devices` does not return any available emulator)